### PR TITLE
Javascript conditionals

### PR DESCRIPTION
**Pull request by RossBrewster**
Screenshot 2023-08-30 at 2 28 56 PM

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+function introduceWarnerBro(name) {
+  switch (name) {
+    case 'yakko':
+      return "We're the Warner brothers!";
+    case 'wakko':
+      return "We're the Warner brothers!";
+    default:
+      return 'Goodnight everybody!';
+  }
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: Missing a `case` clause for `"dot"`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Ross!

**Comment by RossBrewster** [comment]
Screenshot 2023-08-30 at 2 28 56 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**